service/history/queues/executable.go

```diff
@@ -292,8 +293,8 @@ func (e *executableImpl) Execute() (retErr error) {
 			trace.WithAttributes(
 				attribute.Key(telemetry.WorkflowIDKey).String(e.GetWorkflowID()),
 				attribute.Key(telemetry.WorkflowRunIDKey).String(e.GetRunID()),
-				attribute.Key("task-type").String(e.GetType().String()),
-				attribute.Key("task-id").Int64(e.GetTaskID())))
+				attribute.Key("task.type").String(e.GetType().String()),
+				attribute.Key("task.id").Int64(e.GetTaskID())))
```

Comment:
On second glance, most attributes are `.` separated, so might just do that here, too.